### PR TITLE
Migrate org members to TanStack server functions

### PR DESCRIPTION
**Pull request by jeevanpillay**
## Summary
- Move org member list/invite/update/remove/revoke behavior into api/app domain commands plus a thin TanStack server-function adapter.
- Replace org member UI and SignalCreatorAvatar tRPC usage with local TanStack Query helpers keyed by the active Clerk org id.
- Remove the orgMembers tRPC router from the app router and add source tests that ratchet the new boundary.

## Verification
- pnpm --filter @api/app test -- src/__tests__/org-members-domain-commands.test.ts src/__tests__/org-members-tanstack-adapter-source.test.ts
- pnpm --filter @lightfast/app test -- src/__tests__/org-members-queries-source.test.ts src/__tests__/authenticated-routes-source.test.ts
- pnpm typecheck
- pnpm check
- git diff --check
- agent-browser smoke: opened https://auth-org-members-tanstack.lightfast.localhost/acme/settings/members unauthenticated; route loaded with HTTP 200 and Clerk sign-in rendered

## Summary by CodeRabbit

* **New Features**
  * Added TanStack Server endpoints for organization member listing, inviting, role updates, removals, and invitation revocations.
  * Updated the organization members UI to use shared React Query query/mutation helpers scoped to the active organization, including optimistic updates and improved empty-state behavior.

* **Bug Fixes**
  * Improved error toasts to surface expected domain error messages more reliably.

* **Tests**
  * Refactored org-member command tests to execute domain commands directly.
  * Added adapter boundary tests to ensure TanStack wiring and removed legacy routing.

* **Chores**
  * Updated public exports and added a new package export subpath for org-member TanStack integration.

### CODE REVIEW

**Comment by jeevanpillay** [comment]
Addressed the Codex review note in 1577bbc92: org-member TanStack adapter domain errors now keep a DomainError marker, and the shared mutation toast handler surfaces those expected domain messages instead of falling back to the generic unexpected-error copy.

**Comment by jeevanpillay** [comment]
Addressed the concrete CodeRabbit notes in d0fd92d4b: Clerk org member/invitation listing now paginates through all pages, privileged mutation authz coverage now covers invite/update/remove/revoke, and the members list renders an explicit no-org state instead of a disabled-query skeleton. Keeping the source-string tests intentionally as architecture boundary ratchets for this migration slice.

File: apps/app/src/__tests__/org-members-queries-source.test.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+  });
+
+  it("surfaces expected domain errors from TanStack mutations", () => {
+    const source = readFileSync(resolve(appRoot, "src/trpc/react.tsx"), "utf8");
+
+    expect(source).toContain("isExpectedDomainError");
+    expect(source).toContain('error.name === "DomainError"');
+    expect(source).toContain("message = error.message");
+  });
```
[the remark this answers is not in the document]
Comment (jeevanpillay) [reply apps/app/src/__tests__/org-members-queries-source.test.ts:46]: Leaving this as an intentional architecture-boundary ratchet for the migration slice. The behavior is covered by the domain command tests and query helper usage; this specific source test exists to prevent reintroducing org-members tRPC coupling while we are strangling the old surface.
